azext_iot/common/utility.py

```diff
@@ -462,3 +462,8 @@ def is_iso8601_time(self, to_validate: str) -> bool:
             return bool(isodate.parse_time(to_validate))
         except Exception:
             return False
+
+
+def ensure_min_version(cur_ver, min_ver):
+    from packaging import version
```

Comment:
Is the `packaging` package ;) pulled in with setuptools or is this a Core CLI dependency import?

Reply:
Ah, I see - it looks like this `packaging` was a dependency of pytest. I'll get the setuptools version imported, I believe it's 
```
from pkg_resources import packaging
packaging.version.parse(...)
```